src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -75,7 +75,14 @@ def tile(self, tensor_in, repeats):
             TensorFlow Tensor: The tensor with repeated axes
 
         """
-        return tf.tile(tensor_in, repeats)
+        try:
+            return tf.tile(tensor_in, repeats)
+        except tf.python.framework.errors_impl.InvalidArgumentError:
+            shape = tf.shape(tensor_in).numpy().tolist()
+            diff = len(repeats) - len(shape)
+            if diff < 0:
+                raise
```

Comment:
Codecov is reporting this line isn't covered. https://codecov.io/gh/scikit-hep/pyhf/pull/1028/diff Can we hit that too?